### ISSUE TITLE
Check bandpass filter implementation

### ISSUE THREAD

**Issue by pvieten** [CAMI-DKFZ/simpa#93]
Bandpass filter seems to introduce artefacts when applied during reconstruction

**Comment by TomTomRixRix** [comment]
I think the problem is that after performing the inverse FFT we computed the absolute value of the complex data instead of just taking the real part.

So basically changing
```python
torch.abs(torch.fft.ifft(filtered_data_in_fourier_space))
```
into
```python
torch.fft.ifft(filtered_data_in_fourier_space).real
```
in `reconstruction_utils.py` should solve the issue.

I've implemented this change on branch `T93-Bandpassfilter`. Have a look if it solves this issue.

**Comment by kdreher** [comment]
Resolved with 8cc6070550af5f830fa2129ff5288cb16257525b
